src/services/shipment_service.js

```diff
@@ -195,27 +203,26 @@ export default (easypostClient) =>
     }
 
     /**
-     * Get the lowest smartrate of this shipment.
-     * @param {string} id
-     * @param {number} deliveryDays
-     * @param {string} deliveryAccuracy
-     * @returns {Object}
+     * Get the lowest smart rate of a shipment.
```

Comment:
We want these to be `smartrate` single word, this is true of any occurrence that got split here